typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.stories.tsx

```diff
@@ -15,15 +15,16 @@ import {
     Faces as ToroidFaces,
     VertexCount as ToroidVertexCount,
 } from "./test_data/PentagonalToroid";
+import { TGrid3DColoringMode } from "./grid3dLayer";
 
 export default {
     component: SubsurfaceViewer,
     title: "SubsurfaceViewer/Grid3D",
-} as ComponentMeta<typeof SubsurfaceViewer>;
+} as Meta<typeof SubsurfaceViewer>;
 
 type NumberQuad = [number, number, number, number];
 
-const Template: ComponentStory<typeof SubsurfaceViewer> = (args) => (
+const Template: StoryFn<typeof SubsurfaceViewer> = (args) => (
```

Comment:
Keep the old deprecated way. I'm currently updating that more globally, using the StoryObj approach...

Reply:
Reverted.